src/eradius_counter_aggregator.erl

```diff
@@ -37,7 +37,7 @@ start_link() ->
 init([]) ->
     ets:new(?MODULE, [ordered_set, protected, named_table, {keypos, #nas_counter.key}, {write_concurrency,true}]),
     eradius:modules_ready([?MODULE]),
-    erlang:send_after(?INIT_HB, self(), heartbeat),
+    %erlang:send_after(?INIT_HB, self(), heartbeat),
```

Comment:
is that intentional?

Reply:
@RoadRunnr it is - as it calls `eradius_counter:collect` which resets/deletes all the counters and as we move towards prometheus we don't need to do this. It looks like that I just forgot to delete this line from here. Will fix this.

While we're here - do we need in eradius_counter_aggregator at all? I see that it provides almost the same API that `eradius_counter`, just + `heartbeat` and `collect`. Maybe it worth to delete it at all (in separate PR of course) and just update `eradius_counter` with aggregator like API if we will need it?

Reply:
`eradius_counter` is a per Erlang node process while the `eradius_counter_aggregator` would go over all nodes in an Erlang cluster and aggregate the values from all nodes.
Keeping them seperate for e.g. the SNMP exporter, might make sense. But I have not really checked to code.

Reply:
@RoadRunnr I'd suggest to provide configuration option to disable/enable eradius_counter_aggregator at eradius startup. So we will preserve backward compatibility by setting it to `true` by default and disable it everywhere where we'll need. What do you think about this?

Reply:
So I've added `{counter_aggregator, true}` option, so by default behaviour of the aggregator will be as it was before